dfastmi/batch/core.py

```diff
@@ -114,392 +118,411 @@ def batch_mode_core(
     success : bool
         Flag indicating whether the analysis could be completed successfully.
     """
-    Q1 : QRuns
-    apply_q1 : Tuple[bool, bool, bool]
-    Q : Vector
-    apply_q : BoolVector
-
     display = False
     data = DFastMIConfigParser(config)
-    
+
     # check outputdir
-    if rootdir == "":
-        rootdir = os.getcwd()
-    outputdir = data.config_get(str, "General", "OutputDir", rootdir + os.sep + "output")
-    if os.path.exists(outputdir):
-        if display:
-            ApplicationSettingsHelper.log_text("overwrite_dir", dict={"dir": outputdir})
-    else:
-        os.makedirs(outputdir)
-    report = open(outputdir + os.sep + ApplicationSettingsHelper.get_filename("report.out"), "w")
+    rootdir = _get_root_dir(rootdir)
+    outputdir = _get_output_dir(rootdir, display, data)
```

Comment:
display is not needed for passing on, it is always false.

Reply:
I need to discuss with Bert

Reply:
for now keep as is